gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -403,13 +403,11 @@ class VeloxTestSettings extends BackendTestSettings {
   enableSuite[GlutenOrcPartitionDiscoverySuite]
     .exclude("read partitioned table - normal case")
     .exclude("read partitioned table - with nulls")
-    .disableByReason("Blocked by ORC Velox upstream not ready")
```

Comment:
Some ORC tests are disabled with the message `ReaderFactory is not registered for format orc` like [VeloxTestSettings](https://github.com/oap-project/gluten/blob/main/gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala#L361-L368), could you also enable them in this PR?

Reply:
OK, will try it latter